portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Message.path

```diff
@@ -290,7 +290,7 @@
 </tr>
 <tr>
 	<td>SUCCESS_DISMISSIBLE</td>
-	<td>//div[contains(@class,'alert-success')][contains(@class,'alert-dismissible')]</td>
+	<td>//div[contains(@class,'alert-success')][contains(@class,'alert-dismissible')] | //div[contains(@class,'alert-success')][contains(@class,'alert-dismissable')]</td>
```

Comment:
@pedroqueiroz there must be a typo somewhere. You added an "OR" with "|", but the only difference is "dismissable", which is not a word, vs. "dismissible" (a instead of i). Can you please look into this? I know it doesn't make a difference in the UI, but it makes the POSHI code a lot harder to maintain if we add stuff like this for tiny differences.

Reply:
Talked to Chema, he said this typo exists in frontend infrastructure in several places, and they don't plan on fixing it. So this is ok (and probably will be needed for future cases)